opis-manager/src/app/home/home.component.ts

```diff
@@ -32,6 +40,9 @@ export class HomeComponent implements OnInit {
   selectedYear: string;
   selectedTeaching: string;
 
+
+  vCds: number[][] = [[], [], []];
```

Comment:
you don't need to specify an empty content, if you want to make the matrix size **[0][3]** you can just specify in the type.

Reply:
I couldn't make the matrix as you said, TS gave me an error